lib/developer_portal/app/views/developer_portal/shared/_swagger_ui.html.liquid

```diff
@@ -8,17 +8,79 @@
 </div>
 
 <script type="text/javascript">
+  var AUTH_METHOD_USER_KEY = 1;
+  var AUTH_METHOD_ID_KEY_PAIR = 2;
+
+  function getAuthParamsForMethod(authMethod) {
```

Comment:
I don't think this is the kind of logic a provider would want to modify. Maybe we could move to its own js file and include it here. This would also avoid code duplication in `app/views/admin/api_docs/base/swagger.html.erb`. WDYT?

Reply:
Sounds like the proper approach